tests/integration-tests/src/lib.rs

```diff
@@ -250,3 +250,15 @@ impl fmt::Debug for NeverReturnsClientStream {
         write!(f, "TestClientStream catalog")
     }
 }
+
+pub const GOOGLE_V4: SocketAddr = SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(8, 8, 8, 8), 53));
+pub const GOOGLE_V6: SocketAddr = SocketAddr::V6(SocketAddrV6::new(
+    Ipv6Addr::new(0x2001, 0x4860, 0x4860, 0, 0, 0, 0, 0x8888),
+    53,
+    0,
+    0,
+));
+pub const CLOUDFLARE_V4_TLS: SocketAddr =
+    SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(1, 1, 1, 1), 443));
+pub const TEST3_V4: SocketAddr =
+    SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(203, 0, 113, 1), 53));
```

Comment:
It shouldn't matter at all, but in the original tests you converted over, this was 203.0.113.0.

Reply:
Right, thanks for calling that out especially since we've seen that these mass refactorings can obfuscate small issues as happened in #2526. I think this makes more sense given that .0 addresses are typically network identifiers and thus should not be used as individual destination addresses.

https://en.wikipedia.org/wiki/IPv4#First_and_last_subnet_addresses